tests/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest==7.4
+pytest-forked==1.6
+scipy==1.10
```

Comment:
Does it mean that we reply on the exact versions?

Reply:
Yep. IMO we should do more of these to avoid surprises out of the blue.